packages/patternfly-4/react-core/src/components/Modal/Modal.js

```diff
@@ -98,15 +98,15 @@ class Modal extends React.Component {
   render() {
     const { ...props } = this.props;
 
-    if (!canUseDOM) {
+    if (!ExecutionEnvironment.canUseDOM) {
```

Comment:
If you do `document.createElement` in the onComponentMount() lifecycle method, this conditional just becomes `!this.container`